rpc/wrtc_signaling_server.go

```diff
@@ -361,6 +361,8 @@ func (srv *WebRTCSignalingServer) Answer(server webrtcpb.SignalingService_Answer
 	// goroutine that waits for a caller to attempt to establish a connection.
 	if HeartbeatsAllowedFromCtx(ctx) {
 		utils.PanicCapturingGo(func() {
+			// Capture as tests can mutate this value.
+			heartbeatInterval := heartbeatInterval
```

Comment:
Err, this doesn't fix the data race

Reply:
Totally correct; we discussed offline. Apologies for the errant fix there 😮‍💨 . Passed `heartbeatInterval` to the signaling server constructor and used a `const defaultHeartbeatInterval` value instead.